docs/mesos/PrepareHosts.md

```diff
@@ -0,0 +1,44 @@
+# Prepare Hosts for Mesos + Calico Deployment 
+
+## Set & verify fully qualified domain name
+These instructions assume each host can reach other hosts using their fully qualified domain names (FQDN).  To check the FQDN on a host use
+
+    $ hostname -f
+
+Then attempt to ping that name from other hosts.
+
+Also important are that Calico and Mesos have the same view of the (non-fully-qualified) hostname.  Ensure that the value returned by `$ hostname` is unique for each host in your cluster.
+
+## Configure your firewall
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable it completely.  Included in this section is configuration examples for `firewalld`.  If you use a different firewall, check your documentation for how to open the listed ports.
+
+Master node(s) require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| zookeeper    | 2181/tcp          |
+| mesos-master | 5050/tcp          |
+| etcd         | 2379/tcp 4001/tcp |
+| marathon     | 8080/tcp          |
```

Comment:
That sounds good to me. I think we should remove the entries for etcd and marathon in this table and add a very similar disclaimer at the bottom of this section: `If you are running etcd, zookeeper, or Marathon on your Master, be sure to open ....`